external-crates/move/crates/move-analyzer/trace-adapter/src/adapter.ts

```diff
@@ -217,7 +217,7 @@ export class MoveDebugSession extends LoggingDebugSession {
         logger.log(`Launching trace viewer for file: ${args.source} and trace: ${args.traceInfo}`);
 
         try {
-            await this.runtime.start(args.source, args.traceInfo, args.stopOnEntry || false);
+            await this.runtime.start(args.source, args.traceInfo.toString(), args.stopOnEntry || false);
```

Comment:
No clue why as all types seem to be in order, but when this is created from an open trace file, its type here is `Object` and not `String`...